test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -18,20 +18,6 @@ export default class BaseFormObject {
     this.el = new Selector('.o-form').nth(index || 0);
   }
 
-  async setTextBoxValue(name, text) {
-    const element = this.el.find(`input[name="${name}"]`);
-
-    // clear exists text
-    await this.t
-      .selectText(element)
-      .pressKey('delete');
-
-    // type new text
-    if (text) {
-      await this.t.typeText(element, text);
-    }
-  }
```

Comment:
Wasn't removed, moved down a bit (also changed internally) in this file for better grouping.